hack/ci-e2e-kind-ha.sh

```diff
@@ -36,7 +36,7 @@ trap "
 make KUBECONFIG=$KUBECONFIG gardener-up
 
 # run test
-make KUBECONFIG=$KUBECONFIG test-e2e-local
+make KUBECONFIG=$KUBECONFIG test-e2e-local-ha
```

Comment:
Should we set `PARALLEL_E2E_TESTS=10` as well like in https://github.com/gardener/gardener/blob/584e63d5ae7672dd5f95f3af1fdfdcdd881465e0/hack/ci-e2e-kind.sh#L38?

Reply:
Yes, I added it.